php/php-8.2.0-slim/wl-tag.sh

```diff
@@ -0,0 +1 @@
+export WLR_TAG="php/8.2.0-slim"
```

Comment:
Hm, shouldn't we have a single release tag for each version?
I mean, creating a php/8.2.0 tag should release both the slim and full version?

Reply:
True. Fixed!